content/help/ops/security/health-checks-and-mtls/index.md

```diff
@@ -1,8 +0,0 @@
----
```

Comment:
WHen deleting a file, it's a good idea to provide an alias for the dead file so that people with bookmarks to the file will be sent to a related location.

See https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#renaming-moving-or-deleting-pages for details.

Reply:
Somehow i missed the setup of this PR, close this one, and can you review https://github.com/istio/istio.io/pull/3382 instead? Thanks!